lib/Styles/meson.build

```diff
@@ -30,6 +30,34 @@ stylesheet_dark_deps = custom_target(
     install_dir: get_option('datadir') / 'themes' / 'Granite' / 'gtk-4.0'
 )
 
+granite_stylesheet_deps = custom_target(
+    'Granite.scss',
+    input: 'Granite.scss',
+    output: 'granite.css',
+    command: [
+        sassc,
+        sassc_opts,
+        '@INPUT@',
+        '@OUTPUT@',
+    ],
+    install: true,
+    install_dir: get_option('datadir') / 'themes' / 'Granite' / 'gtk-4.0'
+)
+
+granite_stylesheet_dark_deps = custom_target(
+    'Granite-dark.scss',
+    input: 'Granite-dark.scss',
+    output: 'granite-dark.css',
+    command: [
+        sassc,
+        sassc_opts,
+        '@INPUT@',
+        '@OUTPUT@',
+    ],
+    install: true,
+    install_dir: get_option('datadir') / 'themes' / 'Granite' / 'gtk-4.0'
```

Comment:
probably doesn't make sense to install these since they're always loaded by `Granite.init ()`